docs/_includes/generated/task_manager_configuration.html

```diff
@@ -37,6 +37,16 @@
             <td style="word-wrap: break-word;">true</td>
             <td>Enable SSL support for the taskmanager data transport. This is applicable only when the global ssl flag security.ssl.enabled is set to true</td>
         </tr>
+        <tr>
+            <td><h5>taskmanager.debug.additional-logging</h5></td>
```

Comment:
How about changing these to `system-resource-logging`, as Stephan suggested?

Reply:
nothing is actually being logger as far as i can tell, so I would suggest `system-resource-metrics` and `system-resource-metrics-polling-interval`.